feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingViewModel.kt

```diff
@@ -120,6 +121,16 @@ class ConfirmStakingViewModel(
         }
         .asLiveData()
 
+    val unstakingTime = liveData(Dispatchers.Default) {
+        val lockupPeriod = interactor.getLockupPeriodInDays()
+        emit(resourceManager.getQuantityString(R.plurals.staking_main_lockup_period_value, lockupPeriod, lockupPeriod))
+    }
+
+    val eraHoursLength = liveData(Dispatchers.Default) {
+        val hours = interactor.getEra1HoursLength()
+        emit(resourceManager.getQuantityString(R.plurals.common_hours_format, hours, hours))
+    }
```

Comment:
Afaik we dicided to not to use livedata but flow instead (replacement is `flow { emit() }.share()`)

Reply:
Done